patches/tModLoader/Terraria/ModLoader/GlobalItem.cs

```diff
@@ -682,10 +682,20 @@ public abstract class GlobalItem : GlobalType<Item, GlobalItem>
 		/// <br/>This is only called when two items of the same type attempt to stack.
 		/// </summary>
 		/// <returns>Whether or not the items are allowed to stack</returns>
-		public virtual bool CanStackInWorld(Item item1, Item item2) {
+		public virtual bool CanStackInWorld(Item increase, Item decrease) {
 			return true;
 		}
 
+		/// <summary>
+		/// Allows you to make things happen when items stack together.
```

Comment:
Needs an additional comment at the end with a space:
```
///
/// This method is not instanced.
```

Reply:
I added this comment to CanStack, CanStackInWorld and OnStack.  Do you agree with the first two?  I set up OnStack the same way as them, so I'm assuming they are also not instanced.